docker/rootfs/etc/services.d/go2rtc/finish

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/execlineb -S1
+if { s6-test ${1} -ne 0 }
+if { s6-test ${1} -ne 256 }
+
+s6-svscanctl -t /var/run/s6/services
```

Comment:
I'm not sure which version of s6-overlay you're using, but if it's 3+ this line has to be changed to:

```suggestion
/run/s6/basedir/bin/halt
```

Reply:
In this case I am also following what is already done with nginx https://github.com/blakeblackshear/frigate/blob/dev/docker/rootfs/etc/services.d/nginx/finish